dub.json

```
@@ -15,7 +15,11 @@
     "dcd:dsymbol": ">=0.16.0-beta.2 <0.17.0",
     "inifiled": "~>1.3.1",
     "emsi_containers": "~>0.9.0",
-    "libddoc": "~>0.8.0"
+    "libddoc": "~>0.8.0",
+    "dmd": {
+      "repository": "git+https://github.com/dlang/dmd.git",
+      "version": "10eb368c1b8c323e1921e991cb2bd8fce535e9b2"
```

Try:

```suggestion
      "version": "6da6066bf148647a0744622ec5b3f9d446daaa55"
```

https://github.com/dlang/dmd/commit/6da6066bf148647a0744622ec5b3f9d446daaa55